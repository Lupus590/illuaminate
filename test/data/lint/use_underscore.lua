-- config: (lint (only (var:use-discard)))
local _, x = print()

print(_)
x[_] = 0
