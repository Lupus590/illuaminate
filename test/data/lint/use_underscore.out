use_underscore.lua:[4:6-4:7]: Using "discard" variable `_`. [var:use-discard]
   │
 4 │ print(_)
   │       ^
use_underscore.lua:[5:2-5:3]: Using "discard" variable `_`. [var:use-discard]
   │
 5 │ x[_] = 0
   │   ^
No errors and 2 warnings
